src/components/PersistentBanner.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import * as Content from '../content/BannerContent';
+import styles from '../styles/PersistentBanner.module.less';
+import { NavLink } from 'react-router-dom';
+
+export const PersistentBanner: React.FC = () => {
+  return (
+    <div className={styles.banner}>
+      <p>
+        <strong>{Content.blue_text}</strong>{' '}
+        <NavLink to="/resources">
```

Comment:
Can you change this to use the routes constant defined here:

https://github.com/UncountedDeaths/UncountedDeaths.github.io/blob/react_transfer/src/routes.ts#L10

It will be something like `routes.RESOURCES.path`